### DISCUSSION TITLE
Copilot issue

### DISCUSSION THREAD

**Discussion by Yerramilli0429** [community/community#12204] [Copilot Conversations]
Copilot is not working for me now how can I use this?

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by Yerramilli0429** [reply]
1. my account is automatically loging out and copilot is not giving me the suggestions now in my visual studio code[](url)

**Reply by Yerramilli0429** [reply]
<img width="960" alt="copilot" src="https://user-images.githubusercontent.com/95398683/156169347-e3e6c5ee-c046-4de4-b09d-5e20ffbdce04.png">

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select GitHub Copilot from the pulldown
